### PR TITLE
bash script to run configs

### PR DESCRIPTION
**Pull request by akumgill**
* allows us to run inference on multiple configs
* skips regenerating any outputs (i.e. simulations, observations, posteriors, inference, plotting) unless the user specifies with a --regenerate flag
* updating readme with updated running instructions
* all .json files to .gitignore for easier future merges

### CODE REVIEW

File: plot/plotutils.py [hunk 1]
```diff
@@ -87,5 +86,4 @@ def plot_chainconsumer(chains, output_dir, infer_type, true_param_mean=[]):
     for ax in ax_list:
         ax.grid(False)
 
-    plt.savefig(
-        os.path.join(output_dir, f'plot_{infer_type}_cc_{timestamp()}.png'))
+    plt.savefig(os.path.join(output_dir, f'{infer_type}_cc.png'))
```
Comment (cavestruz) [inline plot/plotutils.py:89]: Quick request - can you have all figures saved as both png and pdf?  pdfs are what we'll put into the paper draft, and pngs are easier to thumbnail scroll through

File: scripts/gen_observations.py [hunk 2]
```diff
@@ -23,6 +27,18 @@
 out_rel_path = f'../outputs/observations/{args.obs_id}.{args.num_obs}'
 out_path = os.path.join(script_dir, out_rel_path)
 
+# Checking if observations already exist from an earlier script run
+if (os.path.isfile(os.path.join(out_path, 'drawn_nfw_profiles.npy'))):
+    # Regenerating observations (continuing script)
+    if args.regenerate:
+        print('Overwriting existing observations because of --regenerate flag')
```
Comment (cavestruz) [inline scripts/gen_observations.py:34]: Oh, good call - we may want to regenerate if something went wrong the first time, and this minimizes having to manually delete.

**Review by cavestruz — CHANGES_REQUESTED** [review]
Minor change requested (to create pdfs in addition to pngs -- just add a line)

File: plot/plotutils.py (hunk 1 above)
Comment (akumgill) [reply plot/plotutils.py:91]: added pdf saving as well

**Pull request merged**